src/components/users/forgotPassword/SendEmail.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable class-methods-use-this */
```

Comment:
Change the `onChange` and `onSubmit`  to arrow functions so that you do not bind them. Also, you will have to lose the constructor and declare the state directly.